features/karaf/openhab-tp/src/main/feature/feature.xml

```diff
@@ -72,20 +72,10 @@
 
 	<feature name="openhab.tp-httpclient" version="${project.version}">
 		<capability>openhab.tp;feature=httpclient;version=${jetty.version}</capability>
+		<feature dependency="true">pax-web-jetty-extras</feature>
```

Comment:
This change seems to have broken the Hue binding which is now complaining about a missing ALPN feature.

https://community.openhab.org/t/philips-hue-clip-2-api-v2-discussion-thread/142111/315?u=andrewfg